### ISSUE TITLE
Group muting was added but does not work

### ISSUE THREAD

**Issue by dakira** [signalapp/Signal-iOS#2201]
### Bug description
After it has been requested many times, muting for group conversations was added with one of the last couple releases. Unfortunately I still receive notifications for messages in groups which I muted.

### Device info
**Device**: iPhone 5S
**iOS version**: 10.3.2
**Signal version:** 2.12.2

**Comment by charlesmchen** [comment]
@dakira There is a known bug (which has been fixed) which caused errors to trigger notifications. Do you see notifications for normal messages (texts, attachments, etc.) or errors?

**Comment by dakira** [comment]
I see notifications for normal messages. After a couple of hours of activating the mute, the notifications for new messages in a group stop. Could it be that there's a timezone issue?

**Comment by charlesmchen** [comment]
@dakira how long are you muting for?  When you mute, you have to chose the mute duration.

**Comment by dakira** [comment]
So far I've tried to mute for a day only.

**Comment by michaelkirk** [comment]
@dakira - are you seeing the full text of the message in the notification, or are you seeing a generic "New Message" notification?

Can you attach a screenshot of your Signal notification preferences? In the Signal app, tap the gear to go to settings, then tap "Notifications".

**Comment by dakira** [comment]
@michaelkirk Sorry for replying so late, I'm traveling a lot these days.

For regular notifications I see the sender name only (because I configured that). For muted group notifications I only see a message like "New signal message" (translated from German). So the settings are "Display: only sender name" and "In-app notifications: disabled". I'll spare you a screenshot of my German Signal. ;)

I did some more tests (with different mute durations) and it appears that muting only starts to work after a couple of hours.
